docs/arc-iac-docs/.pages.yml

```diff
@@ -38,6 +38,9 @@ nav:
           - Waf:
               - Waf: "arc-iac-docs/modules/terraform-aws-arc-waf"
               - Module Usage Guide: "arc-iac-docs/modules/terraform-aws-arc-waf/docs/module-usage-guide"
+          - Elasticache:
+              - Elasticache: "arc-iac-docs/modules/terraform-aws-elasticache"
+              - Module Usage Guide: "arc-iac-docs/modules/terraform-aws-elasticache/docs/README.md"
```

Comment:
This is named wrong if this is supposed to be an arc module

Reply:
This is not arc module @tsaucier-sf . We don't have arc module for Elasticache